api/v1beta1/helmchart_types.go

```diff
@@ -32,6 +32,7 @@ type HelmChartSpec struct {
 
 	// The chart version semver expression, ignored for charts from GitRepository
 	// and Bucket sources. Defaults to latest when omitted.
+	// +kubebuilder:default:=latest
```

Comment:
```suggestion
	// +kubebuilder:default:=*
```

Also, the description should be:

> Defaults to latest stable when omitted.

This was uncovered in another PR but not changed yet.